### PR TITLE
Update shoot_autoscaling.md

### PR DESCRIPTION
**Pull request by christianhuening**
added the default value for `ScaleDownUtilizationThreshold`

**How to categorize this PR?**

/area documentation

**What this PR does / why we need it**:
adds default value for `ScaleDownUtilizationThreshold`

### CODE REVIEW

**Comment by ghost** [comment]
Thank you @christianhuening for your contribution. Before I can start building your PR, a member of the organization must set the required label(s) {'reviewed/ok-to-test'}. Once started, you can check the build status in the PR checks section below.

File: docs/usage/shoot_autoscaling.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ The `Shoot` API allows to configure a few flags of the `cluster-autoscaler`:
 * `.spec.kubernetes.clusterAutoscaler.ScaleDownDelayAfterDelete` defines how long after node deletion that scale down evaluation resumes (defaults to `ScanInterval`).
 * `.spec.kubernetes.clusterAutoscaler.ScaleDownDelayAfterFailure` defines how long after scale down failure that scale down evaluation resumes (default: `3m`).
 * `.spec.kubernetes.clusterAutoscaler.ScaleDownUnneededTime` defines how long a node should be unneeded before it is eligible for scale down (default: `30m`).
-* `.spec.kubernetes.clusterAutoscaler.ScaleDownUtilizationThreshold` defines the threshold in % under which a node is being removed.
+* `.spec.kubernetes.clusterAutoscaler.ScaleDownUtilizationThreshold` defines the threshold in % under which a node is being removed (default: `50`).
```
Comment (rfranzke) [inline docs/usage/shoot_autoscaling.md]: Actually, the description is a bit wrong, right? The value is of type `float64` and the default is `0.5`, so it's not really 'percent'

Comment (christianhuening) [reply docs/usage/shoot_autoscaling.md]: yepp

**Review by rfranzke — APPROVED** [review]
/lgtm
/reviewed ok-to-test

**Pull request merged**